datalad/plugin/wtf.py

```diff
@@ -262,6 +293,20 @@ class WTF(Interface):
             config and metadata which could potentially contain sensitive 
             information (credentials, names, etc.).  If 'some', the fields
             which are known to be sensitive will still be masked out"""),
+        sections=Parameter(
+            args=("-S", "--section"),
+            action='append',
+            dest='sections',
+            metavar="SECTION",
+            constraints=EnsureChoice(*sorted(SECTION_CALLABLES)) | EnsureNone(),
+            doc="""section to include.  If not set, all sections. This option
+            can be given more than once to include multiple sections."""),
```

Comment:
This would benefit from `[CMD]` and `[PY]` markers.

Reply:
did for `[CMD` using similar phrasing as found in others